docs/pages/example/mapbox-gl-geocoder-local-geocoder.html

```diff
@@ -57,11 +60,167 @@
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.619185,
+                    41.876367
+                ],
+                "type": "Point"
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "title": "Humboldt Park",
+                "description": "A large park on Chicago's northwest side"
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.70199,
+                    41.905423
+                ],
+                "type": "Point"
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "title": "Douglas Park",
+                "description": "A large park near in Chicago's North Lawndale neighborhood"
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.699329,
+                    41.860092
+                ],
+                "type": "Point"
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "title": "Calumet Park",
+                "description": "A park on the Illinois-Indiana border featuring a historic fieldhouse"
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.530221,
+                    41.715515
+                ],
+                "type": "Point"
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "title": "Jackson Park",
+                "description": "A lakeside park that was the site of the 1893 World's Fair"
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.580389,
+                    41.783185
+                ],
+                "type": "Point"
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "title": "Columbus Park",
+                "description": "A large park in Chicago's Austin neighborhood"
+            },
+            "geometry": {
+                "coordinates": [
+                    -87.769775,
+                    41.873683
+                ],
+                "type": "Point"
+            }
+        }
+    ],
+    "type": "FeatureCollection"
+};
+
+
+
+function forwardGeocoder(query) {
+    var matchingFeatures = [];
+    for (var i = 0; i < customData.features.length; i++) {
+        var feature = customData.features[i];
+        // handle queries with different capitalization than the source data by calling toLowerCase()
+        if (feature.properties.title.toLowerCase().search(query.toLowerCase()) !== -1) {
+            // add a tree emoji as a prefix for custom data results
+            feature['place_name'] = '🌲 ' + feature.properties.title;
```

Comment:
I'd add in a note about this being in the carmen geojson format https://github.com/mapbox/carmen/blob/master/carmen-geojson.md to explain these properties.